src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -33,6 +39,17 @@
     'max': None
 }
 
+BOOTSTRAP_SERVERS = os.environ.get('BOOTSTRAP_SERVERS')
+SASL_PLAIN_USERNAME = os.environ.get('SASL_PLAIN_USERNAME')
+SASL_PLAIN_PASSWORD = os.environ.get('SASL_PLAIN_PASSWORD')
+SECURITY_PROTOCOL = os.environ.get('SECURITY_PROTOCOL', 'SASL_SSL')
+SASL_MECHANISM = os.environ.get('SASL_MECHANISM', 'SCRAM-SHA-512')
```

Comment:
It would be great if you could add some documentation about these variables. For example, what does `BOOTSTRAP_SERVERS` do?